apps/server/test/lexical/server/project/diagnostics/state_test.exs

```diff
@@ -22,8 +22,8 @@ defmodule Lexical.Project.Diagnostics.StateTest do
     Path.join([Project.root_path(project()), "lib", "project.ex"])
   end
 
-  def source_file(contents \\ "") do
-    file_uri = SourceFile.Path.to_uri(existing_file_path())
+  def source_file(contents \\ "", file_path \\ existing_file_path()) do
```

Comment:
i really dislike functions with multiple default arguments, but it turns out that we always set the contents, so you can remove that one and have the file_path be the default.